tests/postgres_tests/test_apps.py

```diff
@@ -27,6 +27,13 @@
 
 @unittest.skipUnless(connection.vendor == "postgresql", "PostgreSQL specific tests")
 class PostgresConfigTests(TestCase):
+    def test_install_app(self):
```

Comment:
We could potentially merge this test with the one below as they're both testing contrib.postgres' ready